addon/helpers/markdown-viewer.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import markdownit from 'markdown-it';
+
+const { htmlSafe } = Ember.String;
+
+export function markdownViewer(params) {
+
+    const [markdown] = params;
+    const html = markdownit({
+        html: false,        // Enable HTML tags in source
+
+        linkify: false,     // Autoconvert URL-like text to links
```

Comment:
Make true by default? I wasn't in any of the conversations about this helper, but from the PR description, looks like we want to allow links?